src/components/modals/CreateRepoModalTemplate.tsx

```diff
@@ -176,7 +191,7 @@ export default function CreateRepositoryModalTemplate(
               label="Repository name"
               isRequired
               fieldId="modal-with-form-form-name"
-              helperTextInvalid="Enter a repository name"
+              helperTextInvalid="Name must be alphanumeric, all lowercase and at most 255 characters long'"
```

Comment:
Can this be updated to include `-` and `_` characters too? The regex supports it, so something like `a-ok` or `a_ok` are valid. Also there's a trailing `'` in this message.

Reply:
Yes, will update it.

Reply:
Oh, and maybe something about starting with an alphanumeric character.